templates/travis/.travis/install.sh.j2

```diff
@@ -44,52 +44,61 @@ else
   TAG=$(git rev-parse --abbrev-ref HEAD | tr / _)
 fi
 
-{% if plugin_name == 'pulpcore' %}
-PLUGIN=pulp_file
-{% else %}
-PLUGIN={{ plugin_name }}
-{% endif %}
-
-# For pulpcore, and any other repo that might check out some plugin PR
-{% for item in additional_plugins %}
+{# For pulpcore, and any other repo that might check out some plugin PR #}
+{%- for item in additional_plugins %}
 if [ -e $TRAVIS_BUILD_DIR/../{{ item.name }} ]; then
   {{ item.name|upper| replace("-", "_") }}=./{{ item.name }}
 else
   {{ item.name|upper| replace("-", "_") }}=git+https://github.com/pulp/{{ item.name }}.git@{{ item.branch }}
 fi
 {% endfor %}
 
-{%- if plugin_name != 'pulpcore' %}if [ -n "$TRAVIS_TAG" ]; then
+{%- if plugin_name != 'pulpcore' %}
+if [ -n "$TRAVIS_TAG" ]; then
   # Install the plugin only and use published PyPI packages for the rest
   cat > vars/vars.yaml << VARSYAML
 ---
 images:
-  - ${PLUGIN}-${TAG}:
-      image_name: $PLUGIN
+  - {{ plugin_name }}-${TAG}:
+      image_name: {{ plugin_name }}
       tag: $TAG
       plugins:
-        - ./$PLUGIN
+        - ./{{ plugin_name }}
         {%- for item in additional_plugins %}
         - {{ item.name }}
         {%- endfor %}
 VARSYAML
 else
-{%- endif %}
   cat > vars/vars.yaml << VARSYAML
 ---
 images:
-  - ${PLUGIN}-${TAG}:
-      image_name: $PLUGIN
+  - {{ plugin_name }}-${TAG}:
+      image_name: {{ plugin_name }}
       tag: $TAG
       pulpcore: ./pulpcore
       plugins:
-        - ./$PLUGIN
+        - ./{{ plugin_name }}
         {%- for item in additional_plugins %}
         - ${{ item.name|upper| replace("-", "_") }}
         {%- endfor %}
 VARSYAML
-{%- if plugin_name != 'pulpcore' %}
 fi
+{%- else -%}
+{#- We are hardcoding pulp_file for pulpcore, but it must still be specified -#}
+{#- under additional_plugins for the loops to work. This is being done       -#}
+{#- for overall code clarity & simplicity.                                    #}
+cat > vars/vars.yaml << VARSYAML
+---
+images:
+  - pulp_file-${TAG}:
+      image_name: pulp_file
+      tag: $TAG
+      pulpcore: ./pulpcore
+      plugins:
+        {%- for item in additional_plugins %}
+        - ${{ item.name|upper| replace("-", "_") }}
+        {%- endfor %}
```

Comment:
Why does pulp_file have to be hardcoded?

Reply:
Ah, I see that we're building this container for pulpcore in order to run the pulp_file tests.

Reply:
Yup.